profiling-ffi/src/exporter.rs

```diff
@@ -79,6 +80,15 @@ pub extern "C" fn ddog_prof_Endpoint_agentless<'a>(
     ProfilingEndpoint::Agentless(site, api_key)
 }
 
+/// Creates an endpoint that writes to a file.
+/// Useful for local debugging.
+/// Currently only supported by the crashtracker.
+/// # Arguments
+/// * `filename` - Path to the output file "/tmp/file.txt".
+#[export_name = "ddog_Endpoint_file"]
+pub extern "C" fn endpoint_file(filename: CharSlice) -> ProfilingEndpoint {
```

Comment:
This should be `ddog_prof_Endpoint_file` to match what other APIs in this file (this was a recent change -- https://github.com/DataDog/libdatadog/pull/359 -- so it was easy to miss)